app/src/main-process/main.ts

```diff
@@ -729,7 +729,13 @@ function createWindow() {
       electron: '>=1.2.1',
     }
 
-    const extensions = [REACT_DEVELOPER_TOOLS, ChromeLens]
+    const axeDevTools = {
+      id: 'lhdoppojpmngadmnindnejefpokejbdd',
+      electron: '>=1.2.1',
+      Permissions: ['tabs', 'debugger'],
```

Comment:
Without this, we get Manifest 3 google chrome extension warnings and the scan feature in this tool fails to run.

Reply:
I believe this allows access to the "tab" that is open.